hw/ip/gpio/dv/cov/gpio_cov_excl.el

```diff
@@ -1,34 +1,59 @@
-// Copyright lowRISC contributors (OpenTitan project).
-// Licensed under the Apache License, Version 2.0, see LICENSE for details.
-// SPDX-License-Identifier: Apache-2.0
```

Comment:
Dropping these lines breaks CI. If you're going to use a tool to generate the file, you'll need to add them back manually.